src/requests/fetchData.js

```diff
@@ -1,12 +1,10 @@
-
-
-export const fetchCountries = async () => {
+export const fetchData = async (endpoint) => {
     let requestOptions = {
         method: 'GET',
         redirect: 'follow'
       }
     try{
-        const response = await fetch(`https://restcountries.eu/rest/v2/all`, requestOptions)
+        const response = await fetch(endpoint, requestOptions)
```

Comment:
Here you could pass in just the end point, so keep the `https://restcountries.eu/rest/v2` and have it like `https://restcountries.eu/rest/v2${endpoint}`